src/components/getComponentsFromType.js

```diff
@@ -14,10 +15,11 @@ import ErrorComponent from './ErrorComponents/ErrorComponent';
 // SimpleSearch components/functions
 import SimpleSearch from './TypeComponents/SimpleSearch/Widget/SimpleSearch';
 import SimpleSearchForm from './TypeComponents/SimpleSearch/Form/SimpleSearchForm';
+import css from './Style.css';
 
 // This function ensures all of the switching logic between differing WidgetTypes happens in a single place,
 // and then passes the relevant components in a bundled object.
-const getComponentsFromType = (widgetType = '') => {
+const getComponentsFromType = (widgetType = '', isLoading) => {
```

Comment:
The code smell says we should have defaulted parameters last. 

I'd like to see instead `isLoading` defaulted to `false`, and ensure that whenever we make use of `isLoading` that it is defaulted to true in the implementing component instead

Ie please change this line to

`const getComponentsFromType = (widgetType = '', isLoading = false) => {`

And ensure that anytime we need it, it is defaulted to true (In useWidgetDefinition we do default it so it is fine in this PR)